src/Temperature_LM75_Derived.h

```diff
@@ -364,6 +394,7 @@ class TI_TMP102_Compatible : public Generic_LM75_9_to_12Bit_OneShot_Compatible {
 #define NXP_PCT2075             Generic_LM75_11Bit
 #define NXP_SE95                Generic_LM75_12Bit
 #define ON_NCT75                Generic_LM75_12Bit
+#define ON_NCT_OS               OnSemi_OneShot
```

Comment:
I think it's fine to, and the precedent here is to override the `Generic*_` classes with the more specific ones. Since your new class is a strict superset of the provided functionality in `Generic_LM75_12Bit`, nothing should be broken by this:

```suggestion
#define ON_NCT75                OnSemi_NCTx75
#define ON_NCT375               OnSemi_NCTx75
```